### PR TITLE
solution

### PR DESCRIPTION
**Pull request by IvanTkachenko3301**

### CODE REVIEW

File: src/convertToObject.js [hunk 1]
```diff
@@ -1,18 +1,32 @@
...
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styleCssObject = {};
+  const fromStringToArray = sourceString
+    .split(';')
+    .map(elem => elem.split(':'))
+    .filter(elem => elem.length !== 1)
+    .map(elem => elem.map(elem2 => elem2.trim()));
```
Comment (Olekk17) [inline src/convertToObject.js:20]: It's been often said that we should not name variables like 'elem', maybe u need to replace it with something with more context

File: src/convertToObject.js [hunk 2]
```diff
@@ -1,18 +1,32 @@
...
+
+  fromStringToArray.forEach(item => {
+    const key = item[0];
+    const property = item[1];
+
+    styleCssObject[key] = property;
+  });
```
Comment (Olekk17) [inline src/convertToObject.js:27]: Learn about Object.fromEntries method:
https://www.programiz.com/javascript/library/object/fromEntries

**Review by Olekk17 — APPROVED** [review]

File: src/convertToObject.js (hunk 1 above)
Comment (liliia-livandovskaya) [reply src/convertToObject.js:20]: Yeap, `elem` is a bad name for the variable

**Review by liliia-livandovskaya — CHANGES_REQUESTED** [review]
Great job! Pay attention to the comments

**Pull request closed without merging**